test-integration/apps/site/gb-tracker-client-3.8.2.js

```diff
@@ -0,0 +1,445 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
We may have been missing a few lines in our `.gitignore`. Let's not check these builds in.